source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -157,12 +160,55 @@ def _do_request(self):
 
         all_free_access = free_access.crowfly | free_access.odt | free_access.free_radius
 
-        # if a place is freely accessible, there is no need to compute it's access duration in isochrone
-        places_isochrone = [p for p in proximities_by_crowfly if p.uri not in all_free_access]
+        access_points_map = defaultdict(list)
 
-        result = {}
+        if self._mode == FallbackModes.car.name or self._request['_access_points'] is False:
+            # if a place is freely accessible, there is no need to compute it's access duration in isochrone
+            places_isochrone = [p for p in proximities_by_crowfly if p.uri not in all_free_access]
+        else:
+            places_isochrone = []
+            for p in proximities_by_crowfly:
+                # if a place is freely accessible, there is no need to compute it's access duration in isochrone
+
+                if p.uri in all_free_access:
+                    continue
+                # what we are looking to compute, is not the stop_point, but the entrance and exit of a stop_point
+                # if any of them are existent
+                if not p.stop_point.access_points:
+                    places_isochrone.append(p)
+
+                if self._direct_path_type == StreetNetworkPathType.BEGINNING_FALLBACK:
+                    for ap in (ap for ap in p.stop_point.access_points if ap.is_entrance):
```

Comment:
So if `ap.is_entrance == False` for all `ap` in `p.stop_point.access_points`, then we will not compute any fallback duration to `p`.
Is it what we want to do ?